python_modules/dagster/dagster/_grpc/impl.py

```diff
@@ -300,6 +304,16 @@ def get_external_sensor_execution(
                     asset_keys=sensor_def.asset_keys,
                 )
             )
+        elif isinstance(sensor_def, FreshnessPolicySensorDefinition):
```

Comment:
I know we didn't start it here, but this is kind of weird that the grpc impl needs to know about the different kinds of sensors.  I would think that we could just pass the repository def to the SensorEvaluationContext in all cases and then have the user-code wrapper within the `FreshnessPolicySensorDefinition` construct the `FreshnessPolicySensorEvaluationContext`.